pyomo/util/units_checking.py

```diff
@@ -0,0 +1,175 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright 2017 National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  __________________________________________________________________________
+#
+#
+""" Pyomo Units Checking Module
+This module has some helpful methods to support checking units on Pyomo
+module objects.
+"""
+from pyomo.core.base.units_container import units, UnitsError, UnitExtractionVisitor
+from pyomo.core.base.objective import Objective
+from pyomo.core.base.constraint import Constraint
+from pyomo.core.base.var import Var
+from pyomo.core.base.param import Param
+from pyomo.core.base.suffix import Suffix
+from pyomo.core.base.set import Set, RangeSet
+from pyomo.gdp import Disjunct
+from pyomo.gdp import Disjunction
+from pyomo.core.base.block import Block
+from pyomo.core.base.external import ExternalFunction
+from pyomo.core.base.expression import Expression
```

Comment:
Would the following be simpler / more "future proof":
```python
from pyomo.core.base import (
    Objective, Constraint, Var, Param, Suffix, Set, RangeSet, Block, 
    ExternalFunction, Expression,
)
from pyomo.gdp import Disjunct, Disjunction
```

Reply:
Sure. I didn't know that this would work.

Reply:
I will make this change too.